asyncscope.md

```diff
@@ -1825,7 +1825,32 @@ spec here
 
 ## `execution::nest()`
 
-spec here
+::: add
+__§34.9.11.14 `std::execution::nest` [exec.nest]__
+
+[1]{.pnum} `nest` nests its input sender within the dynamic lifetime of the async scope referenced by its input async
+scope token.
+
+[2]{.pnum} The name `nest` denotes a customization point object. For subexpressions `sndr` and `token`, let `Sndr` be
+`decltype((sndr))` and let `Token` be `decltype((token))`. If `Sndr` and `Token` do not satisfy
+`async_scope_token<Sender, Token>` then `nest(sndr, token)` is ill-formed.
+
+[3]{.pnum} Otherwise, the expression `nest(sndr, token)` is expression-equivalent to:
+```cpp
+Token{token}.nest(sndr);
```

Comment:
The intention here is that `nest` takes its token argument by value; I think that means I need to express this as `decay_t<Token>{token}`.